### PR TITLE
Add an command to create user

### PR DESCRIPTION
**Pull request by LogischJo**
This PR includes an artisan command to create an admin account mainly for the first installation.

### CODE REVIEW

**Review by RamonRobben — APPROVED** [review]
Looks fine to me. @AVMG20 is testing it so he will merge once approved by him.

**Comment by RamonRobben** [comment]
@LogischJo I changed the branch where you are merging to to the Development branch since we will be using that from now on. So for your future pull requests please merge to the Development branch instead of main.

File: app/Console/Commands/createUser.php
Comment (AVMG20) [inline app/Console/Commands/createUser.php:46]: Can you please check if this password is at least 8 chars :)

File: app/Console/Commands/createUser.php
Comment (AVMG20) [inline app/Console/Commands/createUser.php:55]: unnecessary semicolon at the end ;

File: app/Console/Commands/createUser.php
Comment (AVMG20) [inline app/Console/Commands/createUser.php:46]: Can you please check if this password is at least 8 chars :)

**Review by AVMG20 — CHANGES_REQUESTED** [review]

File: app/Console/Commands/createUser.php
Comment (LogischJo) [reply app/Console/Commands/createUser.php:46]: Do you already have Password Validation rules which I could use?

Comment (AVMG20) [reply app/Console/Commands/createUser.php:46]: you only have to verify if the password is at least 8 long for now :) better password validation can be added later on :)

Comment (RamonRobben) [reply app/Console/Commands/createUser.php:46]: > Do you already have Password Validation rules which I could use?

As far as I believe we do not currently have this. Password just needs to be at least 8 characters.

Comment (LogischJo) [reply app/Console/Commands/createUser.php:46]: k

File: app/Console/Commands/createUser.php
Comment (LogischJo) [reply app/Console/Commands/createUser.php:62]: Removed

File: app/Console/Commands/createUser.php
Comment (LogischJo) [reply app/Console/Commands/createUser.php:46]: Added new Commit

**Pull request merged**
